bundles/admin/admin-layereditor/resources/locale/fi.js

```diff
@@ -34,23 +34,23 @@ Oskari.registerLocalization(
                 },
                 "locale": {
                     "generic": {
-                        "name": "Nimi kielellä {0}",
-                        "subtitle": "Kuvaus kielellä {0}"
+                        "name": "Nimi",
+                        "description": "Kuvaus"
                     },
                     "en": {
                         "lang": "Englanti",
-                        "name": "Nimi englanniksi",
-                        "subtitle": "Kuvaus englanniksi"
```

Comment:
If we remove the change from generic we should remove these language specific definitions since they are the same as in generic. However in swedish these were not removed (its still `Beskrivning på engelska`)?